sdk/confidentialledger/azure-confidentialledger/README.md

```diff
@@ -4,6 +4,40 @@ Azure Confidential Ledger provides a service for logging to an immutable, tamper
 
 [Source code][confidential_ledger_client_src] | [Package (PyPI)][pypi_package_confidential_ledger] | [API reference documentation][reference_docs] | [Product documentation][confidential_ledger_docs]
 
+# Table of Contents
```

Comment:
We generally only havetable of contents for the examples in a readme. That being said, I don't think we're necessarily opposed, just wanted to let you know. Also not sure if it will pass the CI check for readmes

Reply:
We've had the table of contents previously; I just moved it up in the file, so it should still pass the CI check.

Reply:
I realized what was happening - VSCode was automatically adding headings to the "table of contents" when I only wanted to create a bulleted list of links to the examples (as done in other READMEs). I added some configuration to avoid this for non-example headings.